packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -887,3 +889,40 @@ class HeroController extends NavigatorObserver {
     return toHero.child;
   };
 }
+
+/// Enables or disables [Hero]es in the widget subtree.
+class HeroMode extends StatelessWidget {
+  /// Creates a widget that enables or disables [Hero]es.
+  ///
+  /// The [child] and [enabled] arguments must not be null.
+  const HeroMode({
+    Key key,
+    @required this.child,
+    this.enabled = true,
+  }) : assert(child != null),
+       assert(enabled != null),
+       super(key: key);
+
+  /// The subtree to place inside the [HeroMode].
+  final Widget child;
+
+  /// Whether or not [Hero]es are enabled in this subtree.
+  ///
+  /// If true, then the [Hero]es in this subtree will animate as usual.
```

Comment:
nit: this have to be a full sentence.
Maybe combine both sentences.
`If this property is false, the [Hero]es in this subtree will not animate on route changes; Otherwise, they will animate as usual`

Reply:
same here

Reply:
this seems not addressed yet?